peachjam/tasks.py

```diff
@@ -229,17 +229,17 @@ def get_deleted_documents(ingestor_id, range_start, range_end):
 
 
 @background(queue="peachjam", remove_existing_tasks=True)
-def update_user_timelines():
+def update_user_follows():
```

Comment:
It probably won't matter, but renaming these will mean existing tasks with the old name will fail.

Reply:
yeah. we can clean them up after deploying